### PR TITLE
ENH: added vmax to open CHX images(low resolution)

### PR DESCRIPTION
**Pull request by sameera2004**

### CODE REVIEW

**Comment by danielballan** [comment]
This seems like a good time to add vmin as well.

Alternatively, we could accept one argument `imshow_kwargs` that contains all the keys to be passed on to `imshow`. But I think `vmin`, `vmax`, and `cmap` are common enough to warrant special kwargs.

File: xray_vision/mask/manual_mask.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 class ManualMask(object):
     @ensure_ax_meth
-    def __init__(self, ax, image, cmap='gray', undo_history_depth=20):
+    def __init__(self, ax, image, cmap='gray', vmax=None, undo_history_depth=20):
```
Comment (tacaswell) [inline xray_vision/mask/manual_mask.py]: add an `imshow_kwargs` input which lets you pass an arbitrary dict through to imshow.

**Comment by tacaswell** [comment]
Also note that the `AxesImage` instance for the data is availble as `mc.base_image` which you can directly modify to change things like the limits and color map.

**Comment by tacaswell** [comment]
And someplace, maybe in the CrossSection code, we have a snippet that will set the limits to the 5 and 95 percentiles.  Pulling that out as a helper function might be worth doing.

**Pull request merged**